manager/integration/tests/test_zone.py

```diff
@@ -0,0 +1,190 @@
+import pytest
+import time
+
+from common import client # NOQA
```

Comment:
Sort out this import part. I think it's better to follow the schema of the engine integration test.

Reply:
fixed

Reply:
I mean:
```
from common import (
......
)

from test_scheduling import ...
```

Reply:
* According to python style guide
  > https://www.python.org/dev/peps/pep-0008/#imports

  it's recommended to imports to be on separate lines.

* I'll create a separate PR to move `wait_new_replica_ready()` to common.